### PR TITLE
Parallel Write Benchmark: Fix 1D

### PR DESCRIPTION
**Pull request by guj**
Found a bug when computing block offset/count with the multi block per rank option is on.    Fixed.

### CODE REVIEW

File: examples/8a_benchmark_write_parallel.cpp [hunk 1]
```diff
@@ -1004,25 +1005,42 @@
...
~            m_InRankMeshLayout.emplace_back(offset, count);
~        }
~        else
~        {
~            auto res = unitCount - avg * (numPartition - 1);
~            Extent count = {res * m_MinBlock[0]};
             m_InRankMeshLayout.emplace_back(offset, count);
         }
     }
+    */
```
[the remark this answers is not in the document]
Comment (ax3l) [reply examples/8a_benchmark_write_parallel.cpp]: @guj thanks a lot!

Can you please remove the unused code and add a PR description! :)

[the remark this answers is not in the document]
Comment (guj) [reply examples/8a_benchmark_write_parallel.cpp]: OK, done. Thanks

**Review by ax3l — APPROVED** [review]
Thanks a lot!

@guj, as a follow-up PR, can you please add inline comments for the functional parts of this benchmark. Coming back to this now, I find it a bit under-documented and adding good comments for functional blocks and functions would help.

Thanks a lot! :)

**Comment by ax3l** [comment]
@guj before merging this, can you please add a pull-request description what kind of bug in 1D was fixed?

**Comment by guj** [comment]
> @guj before merging this, can you please add a pull-request description what kind of bug in 1D was fixed?

@ax3l done.

**Comment by ax3l** [comment]
Thank you :)

**Pull request merged**